lib/transmuxer/ts_transmuxer.js

```diff
@@ -149,7 +150,8 @@ shaka.transmuxer.TsTransmuxer = class {
       // MediaSource.isTypeSupported('audio/mp4; codecs="mp3"') -> true
       const codecs = shaka.util.MimeUtils.getCodecs(mimeType)
           .replace('mp4a.40.34', 'mp3').split(',')
-          .map(this.getCorrectVideoCodecs_).join(',');
+          .map(shaka.util.StreamUtils.getCorrectAudioCodecs)
```

Comment:
This wasn't mentioned in the commit description. You should probably amend it.

Reply:
I removed the function